src/controllers/HomeController.ts

```diff
@@ -0,0 +1,7 @@
+import { renderTemplate } from '../utils/AppUtils';
+import { Request, Response, NextFunction } from 'express';
+
+export class HomeController {
+
+    public sayHello: (req: Request, res: Response, next: NextFunction) => void = renderTemplate('index');
```

Comment:
I do not find `renderTemplate` to be useful. I suggest we do `res.render('index')`